### PR TITLE
Assign hr to leave application

### PR DESCRIPTION
**Pull request by yusuffGoodie**
## Is this a Feature, Chore or Bug?
- [*] Feature
- [] Chore
- [] Bug

## Analysis and design (optional)
Analyse and attach the design documentation

## Solution description
Describe your code changes in detail for reviewers.

## Is there a business logic within a doctype?
    - [*] Yes
    - [] No

## Output screenshots (optional)
Post the output screenshots, if a UI is affected or added due to this feature.

## Areas affected and ensured
List out the areas affected by your code changes.

## Is there any existing behavior change of other features due to this code change?
Mention Yes or No. If Yes, provide the appropriate explanation.

## Did you test with the following dataset?
- [*] Existing Data
- [] New Data

## Was child table created?
    - [] is attachment required?
        did you test attachment
## Did you delete custom field?
    - [] Yes
    - [*] No
        If yes, did you write a delete patch?

## Is patch required?
- [] Yes
- [*] No
    ## Was the patch test?

## Which browser(s) did you use for testing?
  - [*] Chrome
  - [] Safari
  - [] Firefox

### CODE REVIEW

File: one_fm/one_fm/custom/leave_application.json [hunk 1]
```diff
@@ -86,8 +86,8 @@
    "dt": "Leave Application",
    "fetch_from": null,
    "fetch_if_empty": 0,
-   "fieldname": "custom_propose_to_date",
-   "fieldtype": "Date",
```
Comment (pjamsheer) [inline one_fm/one_fm/custom/leave_application.json]: It seems propose to and from dates are removed from the custom fields

**Review by pjamsheer — CHANGES_REQUESTED** [review]

File: one_fm/one_fm/custom/leave_application.json (hunk 1 above)
Comment (yusuffGoodie) [reply one_fm/one_fm/custom/leave_application.json]: I have resolved this, just that the position the field I added takes is the position where custom_propose_to_date is before. The field wasn't removed

File: one_fm/fixtures/assignment_rule.json [hunk 2]
```diff
@@ -1,132 +1,4 @@
...
-  "document_type": "Shift Request",
-  "due_date_based_on": null,
-  "field": "shift_approver",
-  "is_assignment_rule_with_workflow": 0,
-  "last_user": "abdullah@one-fm.com",
-  "modified": "2024-07-22 13:32:41.603305",
-  "name": "Shift Request Pending Approval",
-  "priority": 0,
-  "rule": "Based on Field",
-  "unassign_condition": "workflow_state in (\"Approved\", \"Rejected\", \"Draft\")",
```
Comment (pjamsheer) [inline one_fm/fixtures/assignment_rule.json]: Why we removed this Assignment Rule

Comment (yusuffGoodie) [reply one_fm/fixtures/assignment_rule.json]: I didn't remove it. I created new assignment rule based on the requirement and I bench export-features to have the changes in my code.

**Review by pjamsheer — APPROVED** [review]

**Pull request merged**